### DISCUSSION TITLE
updateRate property

### DISCUSSION THREAD

**Discussion by Horikaze** [KyryloKuzyk/PrimeTween#84] [Ideas]
For example:
`Tween.Custom(startValue: 0, endValue: 10f, updateRate: 1f, duration: 10f, onValueChange: v => Debug.Log(v));`
prints:
1
2
...
10

**Comment by KyryloKuzyk** [comment]
Thanks for the feature request!


Currently, the closest solution to your request is something like this:
```csharp
float value;

void Example() {
    value = 0f;
    Sequence.Create(10)
        .ChainDelay(1f)
        .ChainCallback(target: this, target => {
            target.value += 1f;
            print(target.value);
        });
}
```

Can you please provide a more detailed real-world example? This will help me better understand the task at hand.

**Reply by Horikaze** [reply]
Yes, this is exactly what I was looking for! It would be nice if this was possible without creating a Sequence, but for now this solution is sufficient.
